src/data/nebulaInterfaces.ts

```diff
@@ -0,0 +1,58 @@
+export interface fetchProfessor_params {
+    firstName: string,
+    lastName: string
+};
+
+export interface fetchCourse_params {
+    subjectPrefix: string,
+    courseNumber: string
+}
+
+export interface requisites {
+    options: Array<any>,
+    required: number,
+    type: string
+}
+
+export interface courseData {
```

Comment:
This can be named just `Course` or `CourseInterface`
This also applies to the rest of the interfaces